pkg/manifests/api/api.go

```diff
@@ -65,13 +66,18 @@ func (mf Manifests) Update() Manifests {
 	return ret
 }
 
-func GetManifests(plat platform.Platform) (Manifests, error) {
+func GetManifests(plat platform.Platform, updaterType string) (Manifests, error) {
 	var err error
 	mf := Manifests{
 		plat: plat,
 	}
 
-	mf.Crd, err = manifests.APICRD()
+	if updaterType == string(updater.NFD) {
+		mf.Crd, err = manifests.APINFDCRD()
+	} else {
+		mf.Crd, err = manifests.APICRD()
```

Comment:
this complicates the test matrix too much. A given release of `deployer` shoudl deploy a single, well defined API version.